account.go

```diff
@@ -409,6 +409,10 @@ func (l *LegalEntity) AppendDetails(values *RequestValues) {
 		values.Add("legal_entity[business_vat_id]", l.BusinessVatID)
 	}
 
+	if l.Verification.Document != nil {
```

Comment:
@brandur I originally did that but `Verification` is not a pointer for some reason and I wasn't sure if there was a "go-like" way to check its presence otherwise.